README.md

```diff
@@ -266,6 +266,81 @@ The side effect of this event is a DPS IEP Review is created which is then writt
 
 - For duplicate NOMIS system generated IEP the only solution is for NOMIS Support Team to delete the rouge IEP records, that is the one or more records that was generated after the DPS one.
 
+### Activities Reconciliation Report Alerts (allocations and attendances)
+
+Daily jobs check each of the prisons which are feature switched on for the Activities service (in the SERVICE_AGENCY_SWITCHES table in NOMIS) and compares the number of allocations and attendances to pay in NOMIS with the number in DPS. When there is a mismatch an alert is sent to the `#sycon-alerts` channel.
+
+Clicking on the `view` button included in the alert will run a query showing all the failures for that day.
+
+#### Allocations reconciliation failures
+
+For allocations a failure indicates that some bookings have a different number of active allocations in NOMIS and DPS. This is a problem that may cause problems with unlock lists in the prison and will likely cause future synchronisation events from DPS to fail.
+
+To try and work out what's different start with the following queries:
+
+NOMIS:
+```sql
+select * from OMS_OWNER.OFFENDER_PROGRAM_PROFILES 
+where OFFENDER_BOOK_ID=<insert offender book id here> 
+and OFFENDER_PROGRAM_STATUS='ALLOC';
+```
+
+DPS:
+```sql
+select * from allocations 
+where booking_id=<insert offender book id here>
+and active=true;
+```
+
+To get an overview of what's been happening with the synchronisation events for this offender run the following query in App Insights:
+```ksql
```

Comment:
hadn't realised that `ksql` was a type - I think you've used `kusto` previously that didn't work

Reply:
I don't think any of them do anything tbh. It's just a clue for the reader.